docs/api.md

```diff
@@ -12,30 +12,27 @@ A [dedicated repository](https://github.com/hugsy/gef-extras) was born to host
 repo is open to all for contributions, no restrictions and the most valuable
 ones will be integrated into `gef.py`.
 
-## Quick start ##
+## Quick start
 
 Here is the most basic skeleton for creating a new `GEF` command named `newcmd`:
 
 ```python
 class NewCommand(GenericCommand):
     """Dummy new command."""
     _cmdline_ = "newcmd"
-    _syntax_  = "{:s}".format(_cmdline_)
+    _syntax_  = f"{_cmdline_:s}"
 
     @only_if_gdb_running         # not required, ensures that the debug session is started
     def do_invoke(self, argv):
-        # do anything allowed by gef, for example show the current running
-        # architecture as Python object:
-        print(" = {}".format(current_arch) )
+        # let's say we want to print some info about the architecture of the current binary
+        print(f"{gef.arch=}")
```

Comment:
TIL! didn't know this cool syntax sugar. I guess it can also be used to greatly improve the readability of pretty printing things like our `GlibcArena` and `GlibcChunk` etc.

Reply:
Good catch, I should use it for the example.